src/config/inotify.c

```diff
@@ -144,3 +154,166 @@ bool check_inotify_event(void)
 
 	return config_changed;
 }
+
+/**
+ * @brief Waits for a specific string to appear in a file, scanning the last N
+ * lines and monitoring for new content.
+ *
+ * This function searches for a given string in the last MAX_LINES lines of the
```

Comment:
It seems you wanted to implement a `MAX_LINES` backwards search but this code is not implemented.

Reply:
Ah, no. I added this in between but removed it in favor of being able to specify *exactly* from which point on we are checking the log. Otherwise, when going back 50 or 100 lines, we may easily catch some "config was reloaded" which is unrelated here but actually came from the preceding test case. I did just fix the comment.